pkgs/pkgs-lib/formats.nix

```diff
@@ -126,27 +126,42 @@ optionalAttrs allowAliases aliases
 
   php = (import ./formats/php/default.nix { inherit lib pkgs; }).format;
 
+  /*
+    Creates a structured value type suitable for serialization formats.
+
+    Parameters:
+    - typeName: String describing the format (e.g. "JSON", "YAML", "XML")
```

Comment:
`nullable` is missing.

Reply:
https://github.com/NixOS/nixpkgs/pull/448871